src/workspacer.Bar/Widgets/BatteryWidget.cs

```diff
@@ -28,23 +32,26 @@ public override IBarWidgetPart[] GetParts()
                 {
                     return Parts(Part(currentBatteryCharge.ToString("#0%"), LowChargeColor));
                 }
+                else if (currentBatteryCharge <= MedChargeThreshold)
+                {
+                    return Parts(Part(currentBatteryCharge.ToString("#0%"), MedChargeColor));
+                }
+                else if (currentBatteryCharge <= HighChargeThreshold)
```

Comment:
Same comment as last time: we don't need high charge threshold, and this else-is block should be merged with the following else-block.